RRS/customer/urls.py

```diff
@@ -10,7 +10,7 @@
         name  = 'index'
     ),
     url(
-        regex = r'restaurant/(\d+)$',
+        regex = r'restaurant/(\d+)/$',
```

Comment:
Please ignore my earlier comment. Adding a trailing slash is the correct way to do it. 

http://www.morethanseven.net/2009/02/10/append-slashes-urls-django/